File: recrutement/src/main/java/com/satge/recrutement/condidature/CondidatureResponse.java
package com.satge.recrutement.condidature;

import lombok.Builder;
import lombok.Getter;

import lombok.Setter;

import java.time.LocalDateTime;
@Getter
@Setter
@Builder
public class CondidatureResponse {
    private Integer id;
    private String titleoffre;
    private String userFirstName;
    private String userLastName;
    private String userEmail;
    private LocalDateTime appliedDate;
    private CondidatureStatus etat;
    private String cvPath;
}

File: OriginalTemplate-master/src/app/services/models/condidature-response.ts
/* tslint:disable */
/* eslint-disable */
export interface CondidatureResponse {
  appliedDate?: string;
  cvPath?: string;
  etat?: 'SOUMIS' | 'EN_REVISION' | 'ENTRETIEN_PROGRAMME' | 'ENTRETIEN_REALISE' | 'ACCEPTE' | 'REFUSE' | 'ARCHIVES';
  id?: number;
  titleoffre?: string;
  userEmail?: string;
  userFirstName?: string;
  userLastName?: string;
}
